lighthouse-core/lib/i18n/i18n.js

```diff
@@ -167,16 +140,35 @@ const formats = {
  * Look up the best available locale for the requested language through these fall backs:
  * - exact match
  * - progressively shorter prefixes (`de-CH-1996` -> `de-CH` -> `de`)
+ * - supported locales in Intl formatters
  *
  * If `locale` isn't provided or one could not be found, DEFAULT_LOCALE is returned.
  * @param {string|string[]=} locales
  * @return {LH.Locale}
  */
 function lookupLocale(locales) {
-  // TODO: could do more work to sniff out default locale
+  // If Node was built with `--with-intl=none`, `Intl` won't exist.
+  if (typeof Intl !== 'object') {
+    throw new Error('Lighthouse must be run in Node with `Intl` support. See https://nodejs.org/api/intl.html for help');
+  }
+
+  // TODO: could do more work to sniff out the user's locale
   const canonicalLocales = Intl.getCanonicalLocales(locales);
 
-  const closestLocale = lookupClosestLocale(canonicalLocales, LOCALES);
+  // Filter by what's available in this runtime.
+  const availableLocales = Intl.NumberFormat.supportedLocalesOf(canonicalLocales);
+
+  const closestLocale = lookupClosestLocale(availableLocales, LOCALES);
+
+  if (!closestLocale) {
+    // Log extra info if we're pretty sure this version of Node was built with `--with-intl=small-icu`.
+    if (Intl.NumberFormat.supportedLocalesOf('es').length === 0) {
+      log.warn('i18n', 'Requested locale not available in this version of node. The `full-icu` npm module can provide additional locales. See https://nodejs.org/api/intl.html#intl_providing_icu_data_at_runtime for help.');
```

Comment:
let's link to https://github.com/GoogleChrome/lighthouse/blob/master/readme.md#how-do-i-get-localized-lighthouse-results-via-the-cli instead?  

This is probably the first error message that node 12 users will see so we'll route them to our FAQ first.